.circleci/config.yml

```diff
@@ -228,9 +228,10 @@ jobs:
           command: |
             mkdir test-results || true
             curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3 -
```

Comment:
@seanmalloy-crest Can you please provide more information on what exact error we have when running `poetry install` without root?

Reply:
If we did not run `poetry install` as root, then the packages would not be installed within the right environment, instead, it would be installed for the current user's (in this case user circleci) environment, and when we would run `sudo poetry run pytest` we would get an error message saying that it could not find the test packages. 
![image](https://user-images.githubusercontent.com/60528866/124199159-29b37900-da87-11eb-96c0-3709bde5d012.png)

Reply:
@seanmalloy-crest You are right.
Let's include a comment in `examples/test_metadata.py` and say that it requires a sudo access to file under `$SPLUNK_HOME/etc/solnlib_demo/metadata/local.meta`.

Reply:
I have added the following comment to examples/test_metadata.py:, feel free to let me know if there are anything else I should add 
`# This test requires sudo permission to access the file under $SPLUNK_HOME/etc/solnlib_demo/metadata/local.meta`